gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineJavaVersionIntegrationTest.groovy

```diff
@@ -341,6 +343,51 @@ class BaselineJavaVersionIntegrationTest extends IntegrationSpec {
         gradleVersionNumber << GRADLE_TEST_VERSIONS
     }
 
+    def '#gradleVersionNumber: jdkToolchainsAutoManagement only sets the language versions'() {
+        when:
+        // language=gradle
+        buildFile << '''
+        javaVersions {
+            libraryTarget = 11
+            runtime = 21
+            jdkToolchainsAutoManagement = true
```

Comment:
This generally looks good, but I thinks this is a bit confusing to me, as you are enabling something which actually ends up disabling code in this plugin (which also happens to be about managing jdkToolchains). I'd probably call this something like `setupJdkToolchains = false` instead.